pkg/controller/cheinstallation/cheinstallation_controller.go

```diff
@@ -158,11 +159,10 @@ func (r *ReconcileCheInstallation) ensureCheNamespace(logger logr.Logger, cheIns
 
 func (r *ReconcileCheInstallation) ensureCheOperatorGroup(logger logr.Logger, ns string, cheInstallation *v1alpha1.CheInstallation) (bool, error) {
 	cheOg := &olmv1.OperatorGroup{}
-	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: ns, Namespace: ns}, cheOg); err != nil {
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: OperatorGroupName, Namespace: ns}, cheOg); err != nil {
```

Comment:
so, this will stay fixed no matter what will be set in the `CheInstallation` resource? I'm just asking to clarify that as from the tests, I got an intention that the OperatorGroup name is based on the Namespace name set in `CheInstallation` resource.

Reply:
well, I believe that's something I got confused with: are the name and the namespace really supposed to be the same?